Traducir.Api/Controllers/StringsController.cs

```diff
@@ -217,5 +217,28 @@ void composePredicate(Func<SOString, bool> newPredicate)
             return BadRequest();
         }
 
+        [HttpDelete]
+        [Authorize(Policy = "CanSuggest")]
+        [Route("app/api/suggestions{suggestionId:INT}")]
+        public async Task<IActionResult> DeleteSuggestion([FromQuery] int suggestionId)
+        {
+            //Get the suggestion
+            var suggestion = await _soStringService.GetStringSuggestionByIdAsync(suggestionId);
```

Comment:
you don't need to call this here... if `DeleteSuggestionAsync` returns `false`, you can then return `BadRequest()`. The goal here is to do everything with only one SQL query :)